src/Compilers/CSharp/Portable/Symbols/Symbol.cs

```diff
@@ -331,6 +334,35 @@ internal virtual LexicalSortKey GetLexicalSortKey()
         /// </summary>
         public abstract ImmutableArray<Location> Locations { get; }
 
+        public virtual bool HasLocationInTree(SyntaxTree tree)
```

Comment:
>public virtual bool HasLocationInTree(SyntaxTree tree) [](http://example.com/codeflow?start=8&length=54)

It looks like this method should be deleted.

Reply:
indeed.